dotCMS/src/main/java/com/dotmarketing/portlets/workflows/model/WorkflowHistory.java

```diff
@@ -65,7 +70,30 @@ public void setCreationDate(Date creationDate) {
     }
 
     public String getChangeDescription() {
-        return changeDescription;
+
+        if (UtilMethods.isSet(this.changeDescription) && this.changeDescription.trim().startsWith("{")) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This call to String.startsWith can be rewritten using String.charAt(0)](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4904110)